src/Page/Guides/Data.elm

```diff
@@ -63,7 +64,24 @@ teaserListFromGuideDict language guides =
         |> List.sortBy .title
 
 
-guideImagefromTeaserImage : Maybe Page.Guide.Data.Image -> Maybe Page.Shared.Data.GuideTeaserImage
+teaserListFromStoryDict :
+    Language
+    -> Page.Story.Data.Stories
+    -> List Page.Shared.Data.Teaser
+teaserListFromStoryDict language stories =
+    Dict.toList (Page.Story.Data.storiesInPreferredLanguage language stories)
```

Comment:
There is a [`Dict.values`](https://package.elm-lang.org/packages/elm/core/latest/Dict#values) that would simplify the code a bit